### DISCUSSION TITLE
I use `<C-w>` in terminal to delete preview word, how to achieve the same behaviour in LazyTerminal?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [LazyVim/LazyVim#1961] [Q&A]
In Neovim, `<C-w>` is a prefix to trigger window related actions, but I don't want this behaviour in terminal mode and want use `<C-w>` to delete the preview word like it did in terminal~

**Comment by dpetka2001** [comment]
If you're in `insert` mode in terminal, you can still use `<C-w>` to delete previous word.

**Reply by LintaoAmons** [reply]
Oh, my fault. There is a hidden keybinding wrote by myself
